python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/mpls-rsvp.j2

```diff
@@ -0,0 +1,94 @@
+{%     if mpls.rsvp.authentication is arista.avd.defined %}
+{%         set auth = mpls.rsvp.authentication %}
+| Authentication type | {{ auth.type | arista.avd.default('-') }} |
+| Authentication sequence-number window | {{ auth.sequence_number_window | arista.avd.default('-') }} |
+| Authentication active index | {{ auth.active_index | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.ip_access_group is arista.avd.defined or mpls.rsvp.ipv6_access_group is arista.avd.defined %}
+| IPv4 access-group | {{ mpls.rsvp.ip_access_group | arista.avd.default('-') }} |
+| IPv6 access-group | {{ mpls.rsvp.ipv6_access_group | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.srlg.strict is arista.avd.defined(true) %}
+| SRLG strict | enabled |
+{%     elif mpls.rsvp.srlg.enabled is arista.avd.defined(true) %}
+| SRLG | enabled |
+{%     endif %}
+{%     if mpls.rsvp.label_local_termination is arista.avd.defined %}
+| Label local-termination | {{ mpls.rsvp.label_local_termination }} |
+{%     endif %}
+{%     if mpls.rsvp.preemption_method.preemption is arista.avd.defined %}
+| Preemption method | {{ mpls.rsvp.preemption_method.preemption }} |
+{%         if mpls.rsvp.preemption_method.timer is arista.avd.defined %}
+| Preemption timer | {{ mpls.rsvp.preemption_method.timer }} |
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.mtu_signaling is arista.avd.defined %}
+| MTU signaling | {{ mpls.rsvp.mtu_signaling | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.fast_reroute is arista.avd.defined %}
+{%         set fast_reroute = mpls.rsvp.fast_reroute %}
+| Fast reroute mode | {{ fast_reroute.mode | arista.avd.default('-') }} |
+| Fast reroute reversion | {{ fast_reroute.reversion | arista.avd.default('-') }} |
+| Fast reroute  bypass tunnel optimization interval | {{ fast_reroute.bypass_tunnel_optimization_interval | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.hitless_restart is arista.avd.defined %}
+| Hitless restart | Active |
+| Hitless restart recovery timer | {{ mpls.rsvp.hitless_restart.timer_recovery | arista.avd.default('-') }} |
+{%     endif %}
+{%     if mpls.rsvp.shutdown is arista.avd.defined(true) %}
+| Shutdown | Active |
+{%     endif %}
+{%     if mpls.rsvp.neighbors is arista.avd.defined %}
+
+##### RSVP Neighbor Authentication
+
+| Neighbor IP | Index | Type |
+| ----------- | ----- | ---- |
+{%         for neighbor in mpls.rsvp.neighbors | arista.avd.natural_sort('ip_address') %}
+| {{ neighbor.ip_address }} | {{ neighbor.authentication.index | arista.avd.default('-') }} | {{ neighbor.authentication.type | arista.avd.default('-') }} |
+{%         endfor %}
+{%         for neighbor in mpls.rsvp.neighbors | arista.avd.natural_sort('ipv6_address') %}
+| {{ neighbor.ipv6_address }} | {{ neighbor.authentication.index | arista.avd.default('-') }} | {{ neighbor.authentication.type | arista.avd.default('-') }} |
+{%         endfor %}
+{%     endif %}
```

Comment:
please fix the natural sort here as ip_address and ipv6_address is not required key

Reply:
Fixed!